plugin/plugin.go

```diff
@@ -114,6 +117,7 @@ func (p *OrmPlugin) Generate(file *generator.FileDescriptor) {
 				if getMessageOptions(msg).GetOrmable() {
 					ormable := NewOrmableType()
 					ormable.Package = fileProto.GetPackage()
+					ormable.OriginName = typeName
```

Comment:
Could pass `OriginName` and `Package` to `NewOrmableType` so all fields are initialized in constructor, as it should be?